app/routes/public/role-invites.js

```diff
@@ -1,51 +1,59 @@
 import Route from '@ember/routing/route';
 
 export default Route.extend({
-  beforeModel(transition) {
+  async routingModel(payload, user, transition) {
+    try {
+      if (this.authManager.currentUser.email === user.email) {
+        await this.loader
+          .post('/role_invites/accept-invite', payload)
+          .then(invite => {
```

Comment:
You're using await, then no need of then

Reply:
@iamareebjamal Okay, I see the error. Fixing it